test/collections/HashDictionaryTest.ooc

```diff
@@ -104,7 +104,7 @@ HashDictionaryTest: class extends Fixture {
 			dictionary add("First", 1337)
 			expect(dictionary getAsType("First", Int) == 1337)
 			expect(dictionary getAsType("First", String) == null)
-			expect(dictionary getAsType("Second", Int) == null)
+			//expect(dictionary getAsType("Second", Int) == null) //FIXME Can we do this without obnoxious Pointer-To-Int warnings?
```

Comment:
`getAsType: func <T>(key: String, T: Class) -> T {`, which means that `dictionary getAsType("Second", Int)` will return an `Int`. If you compare an `Int` to `null`, you will get the warning. If the test would follow the warning `The Class parameter for Covers must be Cell<Cover>` by calling `dictionary getAsType("Second", Cell<Int>)` the warning should go away, but then you also need to use `Cell` when adding things to the dictionary.